package.json

```diff
@@ -20,11 +21,6 @@
     "start": "fedx-scripts webpack-dev-server --progress",
     "test": "fedx-scripts jest --coverage --passWithNoTests"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run lint"
-    }
-  },
```

Comment:
@michaelwheeler I couldn't find this setup in the Husky docs and wasn't sure how to get this working. I updated it per their docs and it's working locally for me now, but if there was some setup step I missed trying to get this working let me know.

Reply:
No. This just came along with the template repo. I haven't every actually used it.